packages/celotool/src/cmds/geth/simulate_client.ts

```diff
@@ -15,6 +15,7 @@ interface SimulateClientArgv extends yargs.Argv {
   recipientIndex: number
   clientCount: number
   reuseClient: boolean
+  dataTest: boolean
```

Comment:
Taking in count that this test is only for "big data tests", why don't rename everything to `bigDataTest`?
It will be clearer in the future if anyone else needs to modify this